buildpacks/frontends/buildpack.toml

```diff
@@ -0,0 +1,33 @@
+# Buildpack API version
+api = "0.11"
+
+# Buildpack ID and metadata
+[buildpack]
+id = "renku/frontends"
+version = "0.0.1"
+name = "Frontend selector buildpack"
```

Comment:
question: I am confused. This buildpack is "renku/frontends" but the name says "Frontend selector" but there is another buildpack in this repo that is also in a folder called "frontend-selector".